### PR TITLE
Update README: add some suitable tags for you.

### PR DESCRIPTION
**Pull request by null09264**

### CODE REVIEW

**Comment by Gisonrg** [comment]
You have misspelled "Standard". Please change it and rebase again.

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# personal
+# Personal Website - A Industrial Standar
```
Comment (Gisonrg) [inline README.md]: Standard

Comment (null09264) [inline README.md]: It is **Standar**, in Chinese it is called “斯坦达”, a synonym for **Titan** or **Giant**

Comment (Gisonrg) [inline README.md]: |                   
                __\--__|_ 
II=======OOOOO[/ ★02 ___|           
          _____\______|/-----. 
        /____________________|              
         \◎◎◎◎◎◎◎◎⊙/ 
           ~~~~~~~~~~~~~~~~

Comment (Gisonrg) [inline README.md]: ···
                        .&______~*@*~______&.           *
                      "w/%%%%%%%%%%%%%%%%%%%\w"        ***
                        `Y""Y""Y"""""Y""Y""Y'         *****
     __/M__          p-p_|__|__|_____|__|__|_q-q      **Y**
____|O_^_O|_________[EEEEM==M==MM===MM==M==MEEEE]-__....|....、
···

**Pull request merged**